client/src/Components/ReviewsChart.jsx

```diff
@@ -1,13 +1,36 @@
 import React from 'react';
-import ChartVisualization from './ChartVisualization.jsx';
+import PropTypes from 'prop-types';
+import ChartVisualization from './ChartVisualization';
 
 import './../styles/ReviewsChart.scss';
-//  __________            .__                    _________ .__                   __   
-//  \______   \ _______  _|__| ______  _  _______\_   ___ \|  |__ _____ ________/  |_ 
+//  __________            .__                    _________ .__                   __
+//  \______   \ _______  _|__| ______  _  _______\_   ___ \|  |__ _____ ________/  |_
 //   |       _// __ \  \/ /  |/ __ \ \/ \/ /  ___/    \  \/|  |  \\__  \\_  __ \   __\
-//   |    |   \  ___/\   /|  \  ___/\     /\___ \\     \___|   Y  \/ __ \|  | \/|  |  
-//   |____|_  /\___  >\_/ |__|\___  >\/\_//____  >\______  /___|  (____  /__|   |__|  
-//          \/     \/             \/           \/        \/     \/     \/             
+//   |    |   \  ___/\   /|  \  ___/\     /\___ \\     \___|   Y  \/ __ \|  | \/|  |
+//   |____|_  /\___  >\_/ |__|\___  >\/\_//____  >\______  /___|  (____  /__|   |__|
+//          \/     \/             \/           \/        \/     \/     \/
+
+/*
+ Static Helper Method
+*/
+const getOverallRatingLabel = percentage => {
+  if (percentage >= 90) {
+    return 'Overwhelmingly Positive';
+  } else if (percentage >= 80) {
+    return 'Very Positive';
+  } else if (percentage >= 70) {
+    return 'Positive';
+  } else if (percentage >= 60) {
+    return 'Mostly Positive';
+  } else if (percentage >= 50) {
+    return 'Mixed';
+  } else if (percentage >= 40) {
+    return 'Mostly Negative';
+  } else if (percentage >= 30) {
+    return 'Negative';
+  }
+  return 'Very Negative';
```

Comment:
Consider using 'else'

Reply:
Linter complained about my else at the end, said it was unnecessary.